app/donu/Schema.hs

```diff
@@ -498,3 +500,36 @@ instance HasSpec ResetExposureStateCommand where
     sectionsSpec "clear-exposure" $
       do reqSection' "command" (jsAtom "clear-exposure-state") "Reset the current session's interpreter state"
          pure $ ResetExposureStateCommand ()
+
+-- | Wrapper for Value to control precisely how these are passed
+-- to clients of Donu
+newtype DonuValue =
+  DonuValue { unDonuDisplay :: Value }
+  deriving Show
+
+instance JS.ToJSON DonuValue where
```

Comment:
Possibly, though my thought here is that there might be a good reason for these to diverge, since this `newtype` wrapper is really to specialize the rendering to Donu's consumers (which is also why I wanted to have the definition in `Donu/` rather than where `Value` is defined)

Reply:
Seems worth having a pointer at least + any notes on why they differ (if necessary)

Reply:
Right, I agree that these should be implemented in different ways. I just want to make sure that if someone adds a case for something in this function, they should remember to also add a corresponding case in `L.A.Exposure.Print`.